lib/components/AttributeItem.jsx

```diff
@@ -95,6 +95,7 @@ class AttributeItem extends React.PureComponent {
         if (this.props.onSetAttributeExpanded) {
             this.props.onSetAttributeExpanded(this.props.item, !this.props.item.expanded);
         }
+        this.onContentClick(e);
```

Comment:
Why is this line needed? There may be a good reason for it, but I could not notice any difference when testing.

Reply:
There was a problem without this line. Without this line, everything worked fine until I tried to click on the 'new descriptor' in 'Sever Setup' after a new descriptor had been added. The editable area did not show up after clicking on the row. The problem is that the 'onExpandAreaClick' function overlaps with the 'onContentClick' function and the 'onContentClick' function is therefore not called here. To solve the issue, I call the 'onContentClick' function insde the 'onExpandAreaClick' function, and everything works fine now.

Reply:
I see. How about `this.selectComponent()` instead of `this.onContentClick(e)`? As it is now, it will call `e.stopPropagation()` twice; first in `this.onExpandAreaClick()` and then in `this.onContentClick()`.

Reply:
Exactly, it called twice. I have updated it.